pkg/testutil/testutil.go

```diff
@@ -83,3 +89,119 @@ func (s Setup) Materialize() (workspaceRoot string, err error) {
 
 	return
 }
+
+var dut = flag.Bool("dut", false, "run command/device under test")
+
+func RunDUT() {
+	if *dut {
+		cmd.Execute()
+		os.Exit(0)
+	}
+}
+
+type CommandFixtureTest struct {
+	Name              string
+	T                 *testing.T
+	Args              []string
+	ExitCode          int
+	NoNestedWorkspace bool
+	StdoutSubs        []string
```

Comment:
This was changed to be a list of strings so I could check for multiple different strings in the output easily.